src/Entity/VotingPlatform/Designation/Designation.php

```diff
@@ -157,7 +157,7 @@ class Designation implements EntityAdministratorBlameableInterface, EntityAdhere
     /**
      * @ORM\Column(type="datetime", nullable=true)
      *
-     * @Groups({"designation_read", "designation_write"})
+     * @Groups({"designation_read"})
```

Comment:
I think we can remove this group entirely 
```suggestion
```

Reply:
we need to display the information somewhere on the front